doc/source/whatsnew/v1.0.0.rst

```diff
@@ -165,7 +165,9 @@ Categorical
 
 - Added test to assert the :func:`fillna` raises the correct ValueError message when the value isn't a value from categories (:issue:`13628`)
 - Bug in :meth:`Categorical.astype` where ``NaN`` values were handled incorrectly when casting to int (:issue:`28406`)
--
+- Using date accessors on a categorical dtyped :class:`Series` of datetimes was not returning the same object that would
```

Comment:
These `:meth:` aren't correct, since they don't resolve to a real method.

How about

```suggestion
- Bug in :class:`Series.str` and :class:`Series.dt` accessor for categorical-dtype returning results with the incorrect length (:issue:`27952`)
```

Reply:
Sure - I've rephrased the first part as you've suggested, but have kept the `Series.dt.tz_localize` example as suggested by @jreback